### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by nadirozsoy** [community/community#41917] [Other Feature Feedback, Questions, & Ideas]

**Comment by DSDmark** [comment]
Hi @nadirozsoym,

check out this:- #18294.

And I suggest you to not create discussion which is already answer by in community. Search first always. And if you have issue with your badge then describe more specifically

**Comment by mohamedzhioua** [comment]
After answering two discussions, You will get Galaxy Brain badge

More details here:
https://github.com/Schweinepriester/github-profile-achievements
